### ISSUE TITLE
Should be able to pre-populate local sync directory with case sensitive path

### ISSUE THREAD

**Issue by MTRichards** [owncloud/client#2753]
As an admin, I want to be able to pre-populate the path for a local sync directory that incorporates the username and the case sensitive name of the server (in this case fictitious HashCloud).

When a user installs the client, it will set up the path properly in the following paradigm:
   "D:/Users/<username>/Desktop/HashCloud" (case is important)

**Comment by danimo** [comment]
Please be more specific as to what the problem is. This should already be possible, assuming a branded client.

**Comment by MTRichards** [comment]
@rperezb if you have any more info on what is not happening.

**Comment by dragotin** [comment]
if the defaultClientFolder in the branding is a relative (not absolute) path, the relative path is prepended by the user home dir as reported by the system, so in the fictuous example above, the setting for defaultClientFolder would simply be
```
QString defaultClientFolder() const { return QLatin1String("HashCloud"); }
```

The problem might be that we can not insert the server name?

**Comment by danimo** [comment]
I think the question here is if its possible to have on the desktop, and it is if a full qualified path is specified.

**Comment by rperezb** [comment]
Yes, the thing is the path, currently, by default the sync folder on the computer is something as: "D:/Users/ownCloud"
What it´s asked for is be possible to modify the folder to:  "D:/Users//Desktop/ownCloud" Including the "Desktop" level.

(It is not about being able to sync just folder on the server side)

**Comment by danimo** [comment]
@rperezb Yes, it is.

**Comment by rperezb** [comment]
It´s not only to add the "Desktop" (or other folder) but the whole path, by default it is used "C:/" instead of "D:/" as requested.

@danimo eager to know how :)
